addons/account_edi_ubl_cii/models/account_edi_common.py

```diff
@@ -532,6 +532,9 @@ def _import_fill_invoice_line_values(self, tree, xpath_dict, invoice_line_form,
         if net_price_unit == 0 and price_subtotal != net_price_unit * (billed_qty / basis_qty) - allow_charge_amount:
             invoice_line_form.price_unit = price_subtotal / billed_qty
 
+        return float_compare(gross_price_unit, net_price_unit,
+                             precision_rounding=self.env.company.currency_id.rounding) != 0
```

Comment:
It may work in some cases, but it's wrong. The net_price_unit = gross_price_unit - rebate. Both the net_price_unit and the gross_price_unit should be tax excluded. Thus, there is no way you can infer the price_include of a tax like this. In addition, this assumes that price_include is the same for every taxes on a given invoice line while you could have many different taxes on a line. We discovered the gross_price_unit was incorrectly computed and this will be corrected in https://github.com/odoo/odoo/pull/105276.